sycl/doc/extensions/proposed/sycl_ext_oneapi_launch_queries.asciidoc

```diff
@@ -125,135 +124,153 @@ class kernel {
   /*return-type*/ ext_oneapi_get_info(T... args) const;
 };
 
-}
+namespace ext::oneapi::experimental::info::kernel {
+
+template <uint32_t Dimensions>
+struct max_work_item_sizes;
```

Comment:
I don't think this needs to be templated on `Dimensions`.  There is already a separate template parameter `Dimensions` on `ext_oneapi_get_info`.

Reply:
Good catch.  I agree this is broken, but I think we need a different fix.  The `info::device::max_work_item_sizes` query in SYCL 2020 _is_ templated, and we should probably be consistent with that.

I think I just screwed this up when defining the overload.  I've converted it to three separate overloads in dcabe369fdf06025a3b301beb5e8848827c94f58.